src/rust/src/lazy/dsl.rs

```diff
@@ -2052,12 +2057,12 @@ impl Expr {
             .into())
     }
 
-    pub fn str_parse_int(&self, radix: Robj) -> Result<Expr, String> {
+    pub fn str_parse_int(&self, radix: Robj, strict: Option<bool>) -> Result<Expr, String> {
         Ok(self
             .0
             .clone()
             .str()
-            .from_radix(robj_to!(Option, u32, radix)?)
+            .from_radix(robj_to!(u32, radix)?, strict.unwrap_or_default())
```

Comment:
@sorhawell I believe this change is due to a change you made upstream. Is this your intended use?

Reply:
I was gonna say I had never seen it in my life! :) 

Now I remember, pretty close, I will change Option<bool> to bool